components/automate-ui/src/app/pages/project/details/project-details.component.html

```diff
@@ -99,10 +99,15 @@
       <ng-container *ngIf="showFirstRuleMessage()">
         <app-authorized [allOf]="['/iam/v2beta/projects', 'post']">
           <div class="empty-case-container">
-            <p>Create the first ingest rule to get started!</p>
-          </div>
-          <div class="empty-case-container">
+            <div class="empty-case-entry">
+              <small *ngIf="project.status === 'EDITS_PENDING'">Deleted rules are pending: update <a [routerLink]="['/settings/projects']">projects</a> to apply edits.</small>
```

Comment:
@tylercloke whoa good catch! could you just make this the same message from the top of the table though?